### ISSUE TITLE
common ants-based approaches to labeling the brain

### ISSUE THREAD

**Issue by satra** [ANTsX/ANTs#587]
for labeling structures in the brain:
1. joint fusion labeling (e.g., with mindboggle atlases)
2. template/label unwarping using antsCorticalThickness outputs (e.g., with mindboggle template +jointfusion atlas)

are there other methods that you have commonly used?

**Comment by gdevenyi** [comment]
https://github.com/cobralab/MAGetBrain - Driven by ANTS underneath, but uses a bunch of minc tools as well
https://github.com/cobralab/antsRegistration-MAGet - Rewritten version driven exclusively by ANTs tools

**Comment by gdevenyi** [comment]
Yup @satra, the rewritten version is input agnostic. Though you can't currently mix types. Its integrated into openneuro already
